lib/main_containers.pm

```diff
@@ -404,5 +389,29 @@ sub load_container_tests {
         }
     }
     loadtest 'containers/bci_logs' if (get_var('BCI_TESTS'));
+
+    ## Helm chart tests. Add your individual helm chart tests here.
+    if (my $chart = get_var('HELM_CHART')) {
+        loadtest 'containers/k3s_ensure_essentials_installed';
```

Comment:
I assume this is temporary until the `k3s` part is done automatically in the container HDD images?